app/utility/fetchSearchData.js

```diff
@@ -0,0 +1,31 @@
+const fetchSearchData = (
+  search = "",
+  page = 1,
+  brands = [],
+  categories = [],
+  colors = [],
+  features = [],
+  tags = [],
+  years = []
+) => {
+  const baseSearchUrl = "https://lolibrary.org/api/search";
+  const data = fetch(`${baseSearchUrl}`, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({
```

Comment:
add documentation to this whole file. need to explain what each of these parameters could be. use jsdoc